tests/sentry/api/endpoints/test_group_user_reports.py

```diff
@@ -36,7 +38,13 @@ def create_environment(self, project, name):
 
     def create_events_for_environment(self, group, environment, num_events):
         return [
-            self.create_event(group=group, tags={"environment": environment.name})
+            self.store_event(
+                data={
+                    "tags": {"environment": environment.name},
+                    "timestamp": iso_format(before_now(seconds=1)),
+                },
+                project_id=self.project.id,
+            )
```

Comment:
In the previous implementation we were creating a events for a specific gorup, now two groups would be created. Does the test still make sense ?

Reply:
Ah, good point. Put them all in the same group.